website/content/docs/getting-started.md

```diff
@@ -175,6 +196,59 @@ The included example topologies:
 * `TaskHookTopology.java` --- A topology that uses a task hook to subscribe to
    event notifications.
 
+## Frequently Asked Questions
```

Comment:
Should we call this section `Troubleshooting`? (cc/ @maosongfu ) Seems like this could grow into it's own page at some point...